### ISSUE TITLE
Application XML Transfer

### ISSUE THREAD

**Issue by WConleyDSM** [DeploymentResearch/HydrationKitWS2022#7]
During the Initial hydration when copying over the Applications (xml control file) - that control file is missing several of the applications. 
So while the source files all copy over the applications are not being populated correctly in the deployment workbench.
All applications for DC01 are there, but the CM01 is missing several applications.

**Comment by AndrewJNet** [comment]
@WConleyDSM Can you provide a list of the missing applications? 

Thanks,
Andrew

**Comment by GeoSimos** [comment]
Here are the ones I have located so far and told my observation to Johan:

1. Configure - Create NO_SMS_ON_DRIVE
2. Configure - WSUS Settings
3. Install - ConfigMgr Reporting Services Point
4. Install - ConfigMgr Software Update Point
5. Install - WSUS
_Also the MDT8456 fix is not implemented._

The CM01 Task Sequence is mostly affected by the above, trying to edit the TS throws a message that there are errors in it and need to be fixed. I fixed the 1st one in the list but the App's GUID is different with that of the original TS.
